README.md

```diff
@@ -2,47 +2,51 @@
 
 The Nabto Edge platform makes it possible to communicate _directly_ between two entities: Instead of interacting indirectly with a device through a cloud service, the platform makes it simple to communicate directly with the actual device to invoke services or transfer data - also through firewalls. Read more on [docs.nabto.com](https://docs.nabto.com/developer/guides/overview/platform-overview.html)!
 
-## Building and Testing
+See the [general introduction](https://docs.nabto.com/developer/platforms/embedded/intro.html) to the Nabto Edge Embedded SDK for an overview and more information.
 
-The Embedded SDK unit tests uses Boost C++ unit tests. If the Embedded SDK is build on a target without a C++ compiler, Cmake will detect the missing compiler and disable building the unit tests. Building of tests can also be disabled manually with `cmake -DDEVICE_BUILD_TESTS=Off -DCMAKE_INSTALL_PREFIX=../_install ..`.
+## Obtaining the Source
+
+The full Nabto Edge Embedded SDK [source code](https://github.com/nabto/nabto-embedded-sdk) is available for simple integration (commercial license).
+
+The Github repo references various 3rd party components as submodules. So remember to clone recursively:
 
-### Building
 ```
-mkdir _build
-cd _build
-cmake -DCMAKE_INSTALL_PREFIX=../_install ..
-cmake --build . --config Release --target install
+git clone --recursive https://github.com/nabto/nabto-embedded-sdk.git
 ```
 
-### Building on Raspberry Pi
+Or download a full source bundle (`nabto-embedded-sdk.zip`) from the [latest release](https://github.com/nabto/nabto-embedded-sdk/releases/latest). This bundle contains all dependencies - and should not be confused with the "Source code" bundle automatically generated by Github that does not contain dependencies.
+
+## Building and Testing
 
-Prerequisite:
-  * ssh access to a Raspberry Pi with Raspberry Pi OS read more here https://www.raspberrypi.com/software/
+The Nabto Edge Embedded SDK can be built using [CMake](https://cmake.org/). The instructions below assume the `cmake` tool is installed and available.
 
-Install tools on the Raspberry Pi
+To build the SDK, a C99 compatible compiler is required (see [Supported environments](#supported-environments)).
 
-```
-sudo apt-get install build-essential cmake git
-```
+To build the SDK's unit tests, a C++ compiler is required. If the SDK is built for a target without a C++ compiler, CMake will detect the missing compiler and disable building the unit tests. Building of tests can also be disabled manually (see below).
```

Comment:
This is false, the auto-detect feature was hacky and did not work properly. It has now been replaced with disabling tests by default. Unittests must now be enabled manually instead.